src/main/java/temaObjContainers/MainClass.java

```diff
@@ -0,0 +1,98 @@
+package temaObjContainers;
+
+import java.util.*;
+
+public class MainClass {
+    public static void main(String[] args) {
+        Persoana pers1 = new Persoana("Andrei", 16);
+        Persoana pers2 = new Student("Razvan", 22);
+        Persoana pers3 = new Angajat("Alexandru", 29);
+        Persoana pers4 = new Somer("Ion", 42);
+
+        System.out.println("\nPersoanele ordonate dupa varsta: ");
+        comparatorVarsta(pers1, pers2, pers3, pers4);
+
+        System.out.println("\nPersoanele ordonate dupa nume: ");
+        comparatorNume(pers1, pers2, pers3, pers4);
+
+        Map<Persoana, List<Hobby>> hobbyPersoane = getPersoanaListMap(pers1, pers2, pers3, pers4);
+
+        System.out.println("\n");
+        afisareHobbySiLocatie(hobbyPersoane, pers1);
+        afisareHobbySiLocatie(hobbyPersoane, pers2);
+        afisareHobbySiLocatie(hobbyPersoane, pers3);
+        afisareHobbySiLocatie(hobbyPersoane, pers4);
+    }
+
+    private static Map<Persoana, List<Hobby>> getPersoanaListMap(Persoana pers1, Persoana pers2, Persoana pers3, Persoana pers4) {
```

Comment:
puteai sa pui in metoda asta si partea de initiailzare a pers1, pers2.... Nu trebuia neaparat sa le pasezi ca param de intrare